README.md

```diff
@@ -1,2 +1,88 @@
 # TicTacToe
 This is the project #3 of the Ruby Curriculum in Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+## Tic Tac Toe
+
+This project consists of a Ruby version of the game [Tic Tac Toe](https://en.wikipedia.org/wiki/Tic-tac-toe).
+
+## Getting Started
+
+### Prerequisites
+To run this project, you need a computer with Ruby installed.
+> If you are uncertain on how to install Ruby, check out this [tutorial](https://www.theodinproject.com/courses/ruby-programming/lessons/installing-ruby-ruby-programming) made by [The Odin Project](https://www.theodinproject.com/about).
+
+### How to Play
+#### In your local enviroment:
+
+1. Clone the repository:
+```
+https://github.com/VanessaAoki/TicTacToe.git
+```
```

Comment:
Please tell your viewers that they have to first **cd** inside the game

Reply:
Thank you for the speedy feedback on this, we will resolve the issues and request another review.